### ISSUE TITLE
Scraping error in game 2008020259

### ISSUE THREAD

**Issue by 903124** [HarryShomer/Hockey-Scraper#18]
Error: 

 ```
File "<string>", line unknown
SelectorSyntaxError: Malformed class selector at position 2
  line 1:
td.+.bborder
  ^
```

**Comment by HarryShomer** [comment]
Hey @903124,

A few things:

1. What version are you running? Running this on the current version yields the following output:
    ```
    Scraping Game  2008020259 2008-11-17
    Warning: Html pbp contains no plays, this game can't be scraped
    ```
    Game 2008020259 happens to be missing the pbp information. It's been like this for as long as      I can remember.

2. If upgrading doesn't work please provide more information on the error (actuals commands you are running, OS, BeautifulSoup version...etc.)

3. As an aside, going forward please provide more details regarding your issue. I know I don't have a template or anything for how issues should be raised but it should go without that what you provided is insufficient. Please respect my time by not making me have to hunt down more details from you.

Best,
Harry S

**Comment by 903124** [comment]
Should have provide more details:

OS: Windows 10
Beautiful soup version: 4.7.1/4.9.1

I've tried to scrape other games with no plays in pbp and it is successful, but fails for games before 2009

**Comment by HarryShomer** [comment]
Thanks for the info!

Just confirmed the issue is caused by the newer versions of BeautifulSoup4. I run an older version on my local environment that doesn't throw this exception. 

Should be easy enough to fix. I'll update you then.

**Comment by HarryShomer** [comment]
@903124 This should be fixed with the most recent commit 4144b5404477b7c9986e44abefcad539e31978c3. I also pushed the fix to [pypi](https://pypi.org/project/hockey-scraper/) so updating the package should resolve any issue.

Thanks for bringing this issue to my attention! Please feel free to reopen this if you feel that it hasn't be resolved.
